Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/demons.yml

```
@@ -0,0 +1,81 @@
+    altDisarm: false
+    soundHit:
+      path: /Audio/Effects/demon_attack1.ogg
+    damage:
+      types:
+        Caustic: 5
+  - type: Damageable
+    damageContainer: BiologicalAdvancedMetaphysical
+    damageModifierSet: AdvancedSpirit
+  - type: NoSlip
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 1
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)